Projects/Faker/Faker/Program.cs

```diff
@@ -1,63 +1,72 @@
 ﻿using System;
-using Faker;
-using Faker.Resources;
-using Newtonsoft.Json;
+using System.Linq;
+using System.Text.Json;
 
-namespace FakerDemo
-{    
-    internal class Program
+for (var i = 0; i < 25; i++)
```

Comment:
I would rather use iteration logic outside the console app and have the console app only return 1 object. But I understand you did this for testing.

Reply:
Correct, it is easy to change this app to return a (json-ed) collection